modules/deploy/pages/deployment-option/cloud/api/cloud-controlplane-api.adoc

```diff
@@ -0,0 +1,189 @@
+= Manage Clusters using the Cloud API
+:description: Use the Control Plane API to manage your clusters and organization-level resources in Redpanda Cloud.
+:page-cloud: true
+:page-beta: true
+:tag-clusters: api:ROOT:cloud-api.adoc#tag--ClusterService
+:tag-networks: api:ROOT:cloud-api.adoc#tag--NetworkService
+:tag-operations: api:ROOT:cloud-api.adoc#tag--OperationService
+:tag-resource-groups: api:ROOT:cloud-api.adoc#tag--ResourceGroupService
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. The Control Plane API enables you to programmatically manage your organization's cloud environment outside of the Redpanda Cloud UI. You can call the API endpoints directly, or use tools like Terraform or Python scripts to automate cluster management.
+
+For the full Cloud API reference documentation, see xref:api:ROOT:cloud-api.adoc[Redpanda Cloud API Reference].
```

Comment:
```suggestion
For the full Cloud API reference documentation, see xref:api:ROOT:cloud-api.adoc[Redpanda Cloud API docs].
```
Again suggesting here to use API docs instead of API reference (although I don't think we have a set standard yet).